test/v1_0_3/non_templating.js

```diff
@@ -4294,7 +4290,7 @@
           .expect(400)
           .end()
           .get(helper.getEndpointStatements() + '?statementId=' + correct.id)
-          .wait(genDelay(stmtTime, '/statmentId=' + correct.id, correct.id))
+          .wait(genDelay(stmtTime, '?statmentId=' + correct.id, correct.id))
```

Comment:
This is just a guess, but shouldn't that read "`statementId`" and not "`statmentId`"?